modules/tensor_mechanics/include/materials/ComputeStressBase.h

```diff
@@ -18,6 +18,13 @@
 
 class ComputeStressBase;
 
+enum class TangentOperatorEnum
+{
+  ELASTIC,
+  LINEAR,
+  NONLINEAR
+};
```

Comment:
It's not immediately obvious to me what the LINEAR option does. It looks like it's only used in ComputeMultiPlasticityStress. Does it make sense for other materials? Since that option will be available for all materials with these changes, we need to make sure it is documented and correctly handled.

Reply:
@bwspenc The linear option is used in ComputeMultiPlasticityStress as an approximation to the full tangent. I was thinking of a) using linear in ComputeMultipleInelasticStress to signify creep only tangent operator, with nonlinear being creep+plastic or b) adding a new enum. I'm not sure which is best? Each model can pick and choose which enum to include from the input file (I think).